src/s1reader/s1_reader.py

```diff
@@ -537,12 +537,18 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
 
         burst_id = f't{track_number:03d}_{id_burst}_{subswath_id.lower()}'
 
-
         #Extract burst-wise information for Calibration, Noise, and EAP correction
         burst_calibration = BurstCalibration.from_calibration_annotation(calibration_annotation,
                                                                          sensing_start)
         burst_noise = BurstNoise.from_noise_annotation(noise_annotation, sensing_start,
                                              i*n_lines, (i+1)*n_lines-1, ipf_version)
+
+        if burst_noise.can_compute_thermal_noise_lut:
+            thermal_noise_lut = burst_noise.compute_burst_thermal_noise_lut((n_lines,
+                                                                             n_samples))
+        else:
+            thermal_noise_lut = None
+
```

Comment:
Due to the comments above, it will calculate LUT regardless even when we do not use it (i.e. Not applying thermal noise correction)
I tested the LUT calculation, and found that it does not take too long time (~0.15 sec. per burst on my intel MBP), but LUT is as large as the burst size in range-doppler coord. (say, ~1500 lines by ~22,000 samples), meaning it will consume about 125MB of memory space. That might not sound big, but considering the case that we load every bursts in a frame (say, 27 bursts in a frame), it will cost ~4GBs of memory. I think it will make the object from `Sentinel1BurstSlc` significantly heavier.